lib/github_changelog_generator/generator/section.rb

```diff
@@ -49,7 +49,7 @@ def get_string_for_issue(issue)
       encapsulated_title = encapsulate_string issue["title"]
 
       title_with_number = "#{encapsulated_title} [\\##{issue['number']}](#{issue['html_url']})"
-      title_with_number = "#{title_with_number}#{line_labels_for(issue)}" if @options[:issue_line_labels].present?
+      title_with_number = "#{title_with_number}#{Entry.new(options).line_labels_for(issue)}" if @options[:issue_line_labels].present?
```

Comment:
Perhaps later, one can build an `@entry` to be used to repeatedly make the `line_labels_for` call on?